test/color/test_gray.py

```diff
@@ -1,4 +1,5 @@
 import pytest
+import cv2
```

Comment:
Please, generate couple of examples with opencv as hardcode them as expected values. 
Dependence on OpenCV is not the best thing :)